### PR TITLE
Add fast sync configuration note

### LINKED ISSUE #1466: Add note about `fast-sync-min-peers`
Add note to set `fast-sync-min-peers` to `1` have to have static peers contribute to the initial sync. See https://github.com/hyperledger/besu/issues/6141 for full context.

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Closes #1466 

I am still investigating . I did not see this option in the ref docs, so added it back from a previous version. Will need to confirm that's correct

### CODE REVIEW

**Review by jflo — APPROVED** [review]

**Comment by joaniefromtheblock** [comment]
This is documented in the CLI command, but not our docs. the `--sync-min-peers` option is documented instead, but this might be helpful for users.

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -4555,6 +4509,52 @@ strict-tx-replay-protection-enabled=false
...
+```bash
+--sync-min-peers=8
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_FAST_SYNC_MIN_PEERS=8
```
Comment (bgravenorst) [inline docs/public-networks/reference/cli/options.md:4535]: Suggestion: BESU_SYNC_MIN_PEERS=8

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -4555,6 +4509,52 @@ strict-tx-replay-protection-enabled=false
...
+
+```bash
+sync-min-peers=8
+```
+
+</TabItem>
+
+</Tabs>
+
+The minimum number of peers required before starting [sync](../../get-started/connect/sync-node.md). The default is 5. Set `fast-sync-min-peers` to `1` to enable static peers to contribute to the initial sync.
```
Comment (bgravenorst) [inline docs/public-networks/reference/cli/options.md:4550]: Suggestion: 
"Set to `1` to enable..."

**Review by bgravenorst — APPROVED** [review]
Minor feedback.

File: docs/public-networks/reference/cli/options.md [hunk 3]
```diff
@@ -4555,6 +4509,52 @@ strict-tx-replay-protection-enabled=false
...
+
+```bash
+sync-min-peers=8
+```
+
+</TabItem>
+
+</Tabs>
+
+The minimum number of peers required before starting [sync](../../get-started/connect/sync-node.md). The default is 5. Set to `1` to enable static peers to contribute to the initial sync.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:4550]:
```suggestion
The minimum number of peers required before starting [sync](../../get-started/connect/sync-node.md). The default is `5`. Set to `1` to enable static peers to contribute to the initial sync.
```

**Pull request merged**